packages/smart-wallet/src/smartWallet.js

```diff
@@ -277,6 +277,7 @@ const SmartWalletKit = defineVirtualFarClassKit(
             balance: a,
           })),
           offerToUsedInvitation: mapToRecord(offerToUsedInvitation),
+          // @ts-expect-error FIXME leftover from offer id string conversion
```

Comment:
I'll fix in https://github.com/Agoric/agoric-sdk/pull/6789. I didn't want to make non-types changes in this PR. This might be better as a TODO comment.